app/assets/stylesheets/patterns/_buttons.scss

```diff
@@ -0,0 +1,16 @@
+.button, input[type="submit"] {
+  padding: 8px 16px;
+  border: none;
+  background: #333;
+  color: #f2f2f2;
+  text-transform: uppercase;
+  letter-spacing: .09em;
```

Comment:
Leading zero?

Reply:
Is it too weird?? I like the letters very close in the buttons :)